nilearn/plotting/displays.py

```diff
@@ -946,6 +1024,134 @@ def draw_cross(self, cut_coords=None, **kwargs):
+                a symmetric matrix.
+            node_coords: numpy array_like of shape (n, 3)
+                3d coordinates of the graph nodes in world space.
+            node_color: color or sequence of colors
+                color(s) of the nodes.
+            node_size: scalar or array_like
+                size(s) of the nodes in points^2.
+            edge_cmap: colormap
+                colormap used for representing the strength of the edges.
+            edge_threshold: str or number
+                If it is a number only the edges with a value greater than
+                edge_threshold will be shown.
+                If it is a string it must finish with a percent sign,
+                e.g. "25.3%", and only the edges with a abs(value) above
+                the given percentile will be shown.
+            edge_kwargs: dict
+                will be passed as kwargs for each edge matlotlib Line2D.
+            node_kwargs: dict
+                will be passed as kwargs to the plt.scatter call that plots all
+                the nodes in one go.
+
+        """
+        # set defaults
+        if edge_kwargs is None:
+            edge_kwargs = {}
+        if node_kwargs is None:
+            node_kwargs = {}
+        if node_color == 'auto':
+            nb_nodes = len(node_coords)
+            node_color = mpl_cm.Set2(np.linspace(0, 1, nb_nodes))
+
+        node_coords = np.asarray(node_coords)
+
+        # safety checks
+        if 's' in node_kwargs:
+            raise ValueError("Please use 'node_size' and not 'node_kwargs' "
+                             "to specify node sizes")
+        if 'c' in node_kwargs:
+            raise ValueError("Please use 'node_color' and not 'node_kwargs' "
+                             "to specify node colors")
+
+        adjacency_matrix_shape = adjacency_matrix.shape
+        if (len(adjacency_matrix_shape) != 2 or
+                adjacency_matrix_shape[0] != adjacency_matrix_shape[1]):
+            raise ValueError(
+                "'adjacency_matrix' is supposed to have shape (n, n)."
+                ' Its shape was {0}'.format(adjacency_matrix_shape))
+
+        node_coords_shape = node_coords.shape
+        if len(node_coords_shape) != 2 or node_coords_shape[1] != 3:
+            raise ValueError(
+                "'node_coords' should be a array with shape (n, 3). "
+                'Its shape was {0}'.format(node_coords_shape))
+
+        if node_coords_shape[0] != adjacency_matrix_shape[0]:
+            raise ValueError(
+                "Shape mismatch between 'adjacency_matrix' "
+                "and 'node_coords'"
+                "'adjacency_matrix' shape is {0}, 'node_coords' shape is {1}"
+                .format(adjacency_matrix_shape, node_coords_shape))
+
+        if sparse.issparse(adjacency_matrix):
+            adjacency_matrix = adjacency_matrix.toarray()
+
+        if not np.allclose(adjacency_matrix, adjacency_matrix.T):
```

Comment:
Please decrease the tolerance here. We don't want a user with code with slight numerical stability to have this code bailing on him: his numerical instabilities are his problem, not ours. I would use an rtol of 1e-3.